src/js/components/Graphs/graph.style.js

```diff
@@ -29,46 +29,127 @@ export const GraphContainer = styled.div`
     }
 `;
 
-export const Graph = styled.div`
-     height: 50%;
+export const GraphContainer = styled.div`
+     height: calc(50% - 87px);
```

Comment:
Why is it `87px`? It looks like a magic constant.

Reply:
GraphHeader height + NavigationButton Height = 87px